src/android/CHIPTool/app/src/main/res/layout/cluster_detail_fragment.xml

```diff
@@ -1,8 +1,51 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
+    android:layout_height="match_parent"
+    android:padding="16dp"
+    android:orientation="vertical">
+  <AutoCompleteTextView
+      android:id="@+id/clusterAutoComplete"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:completionHint="Select a cluster"
```

Comment:
All strings should be in `strings.xml` instead of directly embedded in XML. Same across this file.